src/indra_cogex/sources/cli.py

```diff
@@ -53,71 +71,129 @@ def _iter_resolvers() -> Iterable[Type[Processor]]:
     " and values are dictionaries matching the __init__ parameters for the processor",
 )
 @click.option(
-    "--nodes-path",
-    default=DEFAULT_NODES_PATH,
+    "--skip_failed_processors",
+    is_flag=True,
+    help="If true, doesn't explode on missing files",
```

Comment:
If true, skips processors that are missing required input files without erroring.